src/shared/components/palette/utils/date.pipe.spec.ts

```diff
@@ -51,12 +51,14 @@ describe('DatePipe', () => {
     expect(message).toBe('26 Jul 2017, ' + getExpectedHour(12) + ':09:05 PM');
   });
 
-  it('should render correct date if UTC date in yyyy-mm-ddThh:mm:ss PM format', () => {
+  /* requirement to no longer have effectively changing timezones negates test
```

Comment:
_Question:_
I'm not sure if this comment implies that the test is dead and should be removed (if so, it should be removed rather than commented out) or if it's suggesting that there's a change in the requirements that means this test needs to change too...?

Reply:
Will edit test instead of removing

Reply:
Done